pkg/pdapi/pd_control.go

```diff
@@ -33,6 +33,8 @@ type PDControlInterface interface {
 	GetPDClient(namespace Namespace, tcName string, tlsEnabled bool) PDClient
 	// GetClusterRefPDClient provides PDClient of the tidb cluster.
 	GetClusterRefPDClient(namespace Namespace, tcName string, clusterDomain string, tlsEnabled bool) PDClient
+	// GetPeerPDClient provides PD Client of the tidb cluster from peerURL.
+	GetPeerPDClient(namespace Namespace, tcName string, tlsEnabled bool, peerURL string, peerKey string) PDClient
```

Comment:
`peerKey` should be `peerName`, because it's actually `pdMember.Name`

Reply:
It's compatible with the code above, we may consider abstract these code in the future. For now, it's better to use similar logic and variable rule.

Reply:
I don't see it's necessary to use `key` in the variable name just because it's a map key. BTW there's a `tcName` parameter here, so I suppose we should keep the variable name as close to the current meaning as possible, which is also convenient for future refactoring.